src/cltkv1/nlp.py

```diff
@@ -149,6 +149,8 @@ def analyze(self, text: str) -> Doc:
 
         return doc
 
+    def __call__(self, text : str) -> Doc:
```

Comment:
@free-variation Let me know if I don't understand right:

``` python
>>> cltk_nlp = NLP(language="lat")
>>> cltk_doc_1 = cltk_nlp.analyze(text=get_example_text("lat"))
>>> cltk_doc_2 = cltk_nlp(text=get_example_text("lat"))
>>> cltk_doc_1 == cltk_doc_2
True
```

I feel this strays from the Pythonic "There should be one—and preferably only one—obvious way to do it.". If you mean this as a simple convenience, I am inclined against it. However if there is a special use case you have in mind, please let me know (I am new to this dunder).

Reply:
Well, I take this from the spaCy guy, who often provides various forms of shorthand for most used cases.  That's why the object is made callable.  It seems useful in that it cuts some typing, but it's just syntactic sugar -- if you hate it, let's drop it.

Reply:
Hate is too strong a word! I'm not so much against it, but for positive reasons want to illustrate to our users several key notions of OOP: First one creates an `NLP()` class for a language, then from that object we do some particular task (namely, analyze text). Enforcing and externalizing clarity of process is something that I actually really value for us … so that's my reasoning.

Reply:
Sure, but that can happen by focusing the instructional aspect of the docs on the `analyze` method.  For people who understand what's going on and just want to get on with it quickly, then helper syntax can speed them up.  At least, that's how it is with spaCy.

Reply:
Good point. Let's keep it :)